### PR TITLE
fixes setting sidebar teams link[fixes SCI-2429]

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-2429

### CODE REVIEW

File: app/views/users/settings/_sidebar.html.erb [hunk 1]
```diff
@@ -63,7 +63,12 @@
        <li class="<%= 'active' if on_settings_team_page? %>" >
          <span class="tree-link line-wrap no-indent">
            <% if on_settings_team_page? %>
```
Comment (mlorb) [inline app/views/users/settings/_sidebar.html.erb]: Do we still need this `if... else` sentence, or can we merge this together?

Comment (ZmagoD) [reply app/views/users/settings/_sidebar.html.erb]: I have no idea.

Comment (mlorb) [reply app/views/users/settings/_sidebar.html.erb]: You can give it a try :smiley_cat:

Comment (ZmagoD) [reply app/views/users/settings/_sidebar.html.erb]: it has to be like this

**Pull request merged**